cmds/build.js

```diff
@@ -1,23 +1,45 @@
 'use strict'
+
+/**
+ * @typedef {import("yargs").Argv}Argv
```

Comment:
```suggestion
 * @typedef {import("yargs").Argv} Argv
```

Should not linter be reporting this ?